assets/xml/interface/kanji.xml

```diff
@@ -0,0 +1,3989 @@
+<Root>
+    <File Name="kanji">
+        <!-- The four-digit number after the prefix is the character's codepoint: the hexadecimal short it corresponds to in the text data -->
+
+        <!-- NonKanji -->
+        <Texture Name="gMsgKanji8140_000000" OutName="msg_kanji_8140_000000" Format="i4" Width="16" Height="16" Offset="0x0"/>
+        <Texture Name="gMsgKanji8141_000080" OutName="msg_kanji_8141_000080" Format="i4" Width="16" Height="16" Offset="0x80"/>
+        <Texture Name="gMsgKanji8142_000100" OutName="msg_kanji_8142_000100" Format="i4" Width="16" Height="16" Offset="0x100"/>
```

Comment:
It took me a bit to realize the second set of values are the offset in the kanji segment itself. What do you think about adding a note explaining it?

Reply:
Done, also added a todo to remove the comment when the textures are properly named without the offset.